static_report/src/components/SingleReport/SingleReport.tsx

```diff
@@ -60,7 +60,7 @@ export default function SingleReport({ data, name }: Props) {
 
   const overview = getReportAsserationStatusCounts(
     table?.assertion_results
```

Comment:
I don't think we should use a ternary operator here anymore `a ? b : c`. 
the method argument should be changed
```ts
export function getReportAsserationStatusCounts(
  assertion: PipeRiderAssertionResult | DbtAssertionResult | null,
): ReportAsserationStatusCounts {  /* .. */ }
```

Reply:
For the assertion type, we can fix it after your #253 due to schema renaming.